Fixed Initial Window State and Added Vibration Disable Hotkey.

RaymondKirk opened the pull request:
Vibration disable for 3rd party controller cables, see https://forums.xbox.com/en-US/thread/0C5F2C1B-82C3-43CD-A398-6095A0E31838.

sighol commented on Windows/Gopher/Gopher/Gopher.cpp:
```diff
@@ -82,6 +86,9 @@ void Gopher::loop() {
 	}
 
+	if (CONFIG_DISABLE_VIBRATION)
+		_vibrationDisabled = !_vibrationDisabled;
```
Add brackets around the if statement body. 
Also, I think you need to add a sleep command of some sorts here. I haven't tested it, but If you hold down the *vibration disable* button, it will toggle vibration on and off at the same frequency as the mouse movement (144 times each second). 

Please correct me if I'm wrong

sighol commented on Windows/Gopher/Gopher/Gopher.cpp:
```diff
@@ -171,33 +182,35 @@ void Gopher::handleDisableButton()
+	setWindowVisibility(_hidden, true);
+}
 
+void Gopher::setWindowVisibility(const bool &hidden, const bool& verbose) const
```
I don't like this `verbose` flag. I think it is more *clean* to remove the verbose flag, and leave the function so that it does only *one* thing. Now it is doing two things: setting window visibility **and** printing to console. 

To keep the functionality, you could add the printing code in the `toggleWindowVisiblity` function.

RaymondKirk commented:
Thank you for the comments! All are valid points, I've added the changes you requested and cleaned up the code slightly. I've also changed the vibration disable hotkey to use the "Start" button as I thought it would make sense if all of the configuration shortcuts have the same modifier key! e.g. Start + Something = a config change.

sighol commented on Windows/Gopher/Gopher/Gopher.cpp:
```diff
@@ -86,8 +86,8 @@ void Gopher::loop() {
 
-	if (CONFIG_DISABLE_VIBRATION)
-		_vibrationDisabled = !_vibrationDisabled;
+	//Vibration
```
This comment is a bit redundant, don't you think?

sighol commented on Windows/Gopher/Gopher/Gopher.cpp:
```diff
@@ -190,28 +190,29 @@ void Gopher::handleDisableButton()
-	}
+	HWND hWnd = GetConsoleWindow();
+	ShowWindow(hWnd, _hidden ? SW_HIDE : SW_SHOW);
 }
```
I like this change a lot!
